AdaptiveCards/resources/future.md

```diff
@@ -12,34 +12,8 @@ While we have made excellent progress defining adaptive cards, there is still lo
 
```

Comment:
Do we want to update this line as well? I think we're past "Made excellent progress defining adaptive cards", and we might want to call out different partners here.

Reply:
Good point; I've removed that section altogether. We already have a Partners page right above that page so we could add in a link again to that page for status -  I thought that redundant for now though and skipped it but let me know if you feel we should have an explicit partner call out or an alternate suggestion ?